Client/Frontend/Menu/AppMenuConfiguration.swift

```diff
@@ -189,6 +193,10 @@ extension AppMenuConfiguration {
         return AppMenuItem(title: SetHomePageTitleString, action: MenuAction(action: AppMenuAction.SetHomePage.rawValue), icon: "menu-Home", privateModeIcon: "menu-Home-pbm", selectedIcon: "menu-Home-Engaged")
     }
 
+    private static var SharePageMenuItem: MenuItem {
+        return AppMenuItem(title: SharePageTitleString, action: MenuAction(action: AppMenuAction.SharePage.rawValue), icon: "bottomNav-send", privateModeIcon: "bottomNav-send", selectedIcon: "bottomNav-sendEngaged")
```

Comment:
I think we'll want to use a different icon here. Looking at "Share" in the menu, it's a bit blurrier, a bit taller, and a slightly lighter shade of gray than the other menu items:

<img width="188" alt="screen shot 2016-05-16 at 11 44 13 am" src="https://cloud.githubusercontent.com/assets/1093266/15300012/8738104a-1b5b-11e6-83fd-1dfad56fdfa4.png">

Also, we'll want to use separate assets for normal vs private browsing, since it currently looks like this:
<img width="190" alt="screen shot 2016-05-16 at 11 44 58 am" src="https://cloud.githubusercontent.com/assets/1093266/15300043/b0e458ea-1b5b-11e6-81cf-0815df04a60d.png">